app/routers/users.py

```diff
@@ -11,15 +13,58 @@
 from fastapi import APIRouter
 from fastapi import Depends
 from fastapi import Form
-from fastapi import HTTPException
+from fastapi import Request
+from fastapi.responses import HTMLResponse
+from fastapi.templating import Jinja2Templates
 from sqlalchemy.orm import Session
 
 router = APIRouter()
 
+templates = Jinja2Templates(directory="app/templates")
+
+
+@router.get("/login", response_class=HTMLResponse)
+async def read_login_form(request: Request):
+    return templates.TemplateResponse("login_form.html", {"request": request})
+
+
+@router.get("/create_account", response_class=HTMLResponse)
+async def read_create_account_form(request: Request):
+    return templates.TemplateResponse("create_account_form.html", {"request": request})
+
+
+@router.post("/create_account/submit", response_class=HTMLResponse)
+async def create_user_endpoint(
+    request: Request,
+    username: str = Form(...),
+    password: str = Form(...),
+    email: str = Form(...),
+    db: Session = Depends(get_db),
+) -> Any:
```

Comment:
반환타입은 엄밀하게는 [`_TemplateResponse`](https://github.com/encode/starlette/blob/9f16bf5c25e126200701f6e04330864f4a91a898/starlette/templating.py#L28C7-L28C24)가 되어야 하나,
이를 상속된 클래스 HTMLResponse로 처리해도 될 것 같습니다.

`Any`는 너무 범위가 넓으니 `HTMLResponse`로 부탁드립니다.

Reply:
지난번에 말씀드린 내용 링크를 찾았습니다. 
FastAPI에서는 reture typer과 reseponse_class 둘 중 하나를 사용하는것으로 보이며, reseponse_class를 사용하면 return type은 Any로 설정하는 예시를 보여주고 있습니다. 저번에 말씀못드린게 이 부분입니다.

'There are some cases where you need or want to return some data that is not exactly what the type declares.
For example, you could want to return a dictionary or a database object, but declare it as a Pydantic model. This way the Pydantic model would do all the data documentation, validation, etc. for the object that you returned (e.g. a dictionary or database object).
If you added the return type annotation, tools and editors would complain with a (correct) error telling you that your function is returning a type (e.g. a dict) that is different from what you declared (e.g. a Pydantic model).
In those cases, you can use the path operation decorator parameter response_model instead of the return type.'

link: https://fastapi.tiangolo.com/tutorial/response-model/#response_model-or-return-type

Reply:
우선은 저번 멘토링 내용대로 return type을 없애고 진행하겠습니다.